index.html

```diff
@@ -0,0 +1,347 @@
+                    javaScript
+                </button>
+            </li>
+        </ul>
+        <div class=" btn see-more">
+
+            <button class="  text-1">
+                See Project
+            </button>
+
+        </div>
+
+    </section>
+    <!-- card four -->
+    <section class="card-info">
+        <div class="image">
+            <img src="images/Snapshoot Portfolio 4.png" alt="">
+        </div>
+        <div class="years">
+            <h1 class="project-title">
+                Multi-Post<br> Stories
+            </h1>
+            <div class="items">
+                <h2 class="canopy">CANOPY</h2>
+                <ul class="experience">
+                    <li>
+                        Back End Dev
+                    </li>
+                    <li>
+                        2015
+                    </li>
+                </ul>
+
+            </div>
+
+        </div>
+        <p class="desc">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <ul class="skills">
+            <li>
+                
+                <button class="text">
+                    html
+                </button>
+            </li>
+            <li>
+                <button class="text">
+                    css
+                </button>
+            </li>
+            <li>
+                <button class="text">
+                    javaScript
+                </button>
+            </li>
+        </ul>
+        <div class=" btn see-more">
+
+            <button class="  text-1">
+                See Project
+            </button>
+
+        </div>
+
+    </section>
+    <section class="About-myself">
+        <h1 class="intro">
+            About
+            <br>Myself
+        </h1>
+        <p class="about-1">
+            Hello I’m a software developer! I can help <br> you build a product , feature or website <br> Look through
+            some of my work and <br> experience! If you like what you see and <br> have a project you need coded, don’t
+            <br> hestiate to contact me.
+        </p>
+        <div class="socials">
+            <h2 class="social-links">LET'S CONNECT</h2>
+            <div class="icons">
+                <ul class="social-media">
+                    <li>
+                        <a href=""> <i class="fa-brands fa-twitter fa-1x"></i></a>
+                    </li>
+                    <li>
+                        <a href=""> <i class="fa-brands  fa-linkedin fa-1x "></i></a>
+                    </li>
+                    <li>
+                        <a href=""> <i class="fa-brands fa-github fa-1x"></i></a>
+                    </li>
+                    <li>
+                        <a href=""> <i class="fa-brands fa-m fa-1x"></i></a>
+                    </li>
+                    <li>
+                        <a href=""><i class="fa-solid fa-hand-peace fa-1x"></i></a>
+                    </li>
+                </ul>
```

Comment:
- [ Optional] You have the same issue here, the social media icons have a different color. Kindly consider changing it.

Reply:
okay. thank you